File: src/main/java/post/Box3.java
package post;

public class Box3 extends Box {
    static float length = 23f, width = 14f, height = 13f;

    public Box3() {
        super();
    }

    @Override
    public boolean validate(float length, float width, int height) {
        return !(this.length < length) && !(this.width < width) && !(this.height < height);
    }
}

File: src/main/kotlin/post/Box3.kt
package post

class Box3 : Box() {

    companion object {
        @JvmStatic
        var boxLength: Float = 23f
        var boxWidth: Float = 14f
        var boxHeight: Int = 13
    }

    override fun validate(length: Float, width: Float, height: Int): Boolean =
        !(boxLength < length || boxWidth < width || boxHeight < height)
}